examples/hello_LoRa/hello_LoRa.ino

```diff
@@ -31,6 +32,9 @@ unsigned char loraData[11] = {"hello LoRa"};
 // How many times data transfer should occur, in seconds
 const unsigned int sendInterval = 30;
 
+// In which port we send data? [TTN supports 0-255, 1 is default]
+uint8_t framePort = 1;
```

Comment:
`framePort` is optionally set on Line 72/73. This should be moved to the `sendData` on Line 73:

```
  // Optionally set the Frame Port (1 to 255)
  // uint8_t framePort = 1;
  // lora.sendData(loraData, sizeof(loraData), lora.frameCounter, framePort);

```